oxd-server/src/main/java/org/xdi/oxd/server/op/GetTokensByCodeOperation.java

```diff
@@ -97,8 +97,9 @@ public IOpResponse execute(GetTokensByCodeParams params) throws Exception {
             return opResponse;
         } else {
             LOG.error("Failed to get tokens because response code is: " + response.getScope());
```

Comment:
It seems we have typo here. Please change it to `response.getStatus` since I guess we want to print status, not scopes here.